packml_ros/src/packml_ros.cpp

```diff
@@ -41,6 +41,7 @@ void PackmlRos::spin()
   while(ros::ok())
   {
     spinOnce();
+    ros::Duration(0.001).sleep();
```

Comment:
Any reason not to use a `ros::Rate` here?

Reply:
That's my bad...I suggested `ros::Duration`.

Reply:
Might even make sense to just call `ros::spin()` here. Or do you want to do some work in the rest of the `while-loop`?

Reply:
Unless I'm mistaken, it seems like the loop only really sleeps and calls `ros::spinOnce()`:

https://github.com/ros-industrial-consortium/packml/blob/df2ae7a23bc60b11eb8fcea91a343fcfbc1ae8b8/packml_ros/src/packml_ros.cpp#L39-L47

Reply:
O, right. I completely missed that.

Might be worthwhile to add a `this->` there then.